packages/core/parcel-bundler/src/Bundler.js

```diff
@@ -142,11 +142,11 @@ class Bundler extends EventEmitter {
       detailedReport: options.detailedReport || false,
       global: options.global,
       autoinstall:
-        typeof options.autoInstall === 'boolean'
-          ? options.autoInstall
+        typeof options.autoinstall === 'boolean'
```

Comment:
This would be a breaking change.
We should check both `options.autoInstall` and `options.autoinstall` or set `options.autoInstall = options.autoinstall` in the cli

Reply:
Thanks for you fast response @DeMoorJasper. 
I prefer the latter, this way we don't need to change the logic of the actual Bundler. I have made the changes.
I am however having issues with a test that isn't related to the work I have been doing, hence the failing tests. Is this a known issue?